Source/SVGKImage.h

```diff
@@ -111,7 +111,6 @@ typedef void (^SVGKImageAsynchronousLoadingDelegate)(SVGKImage* loadedImage);
 + (SVGKImage*) imageWithSource:(SVGKSource *)newSource; // if you have custom source's you want to use
 
 - (id)initWithContentsOfFile:(NSString *)path;
-- (id)initWithData:(NSData *)data;
```

Comment:
I can add the empty method body as your comment. Thanks.
By the way, how to get the data of SVGKImage object? And I think we do not need cache it which come from SVG file or has large content.